assembl/static2/js/app/components/debate/brightMirror/sideComment/sideCommentBox.jsx

```diff
@@ -255,83 +307,49 @@ class DumbSideCommentBox extends React.Component<Props, State> {
-              <div className="extracts-nb-msg">
-                <Translate value="debate.brightMirror.commentersParticipation" count={extracts.length} />
-              </div>
-            )}
-            <div className="profile">
-              <AvatarImage userId={userId} userName={userName} />
-              <div className="harvesting-infos">
-                <div className="username">{userName}</div>
-                {!submitting &&
-                  currentComment &&
-                  currentComment.creationDate && (
-                    <div className="harvesting-date" title={currentComment.creationDate}>
-                      {moment(currentComment.creationDate)
-                        .locale(contentLocale)
-                        .fromNow()}
-                    </div>
-                  )}
-                {submitting && (
-                  <div className="harvesting-date">
-                    <Translate value="harvesting.now" />
-                  </div>
-                )}
-              </div>
+          {!submitting && (
+            <div className="extracts-nb-msg">
+              <Translate value="debate.brightMirror.commentersParticipation" count={extracts.length} />
             </div>
-          </div>
-          <div className="harvesting-box-body">
-            {!submitting &&
-              extracts && (
-                <div className="body-container">
-                  <div className="previous-extract">
-                    {extractIndex > 0 && (
-                      <div
-                        onClick={() => {
-                          this.changeCurrentExtract(-1);
-                        }}
-                      >
-                        <span className="assembl-icon-down-open grey" />
-                      </div>
-                    )}
-                  </div>
-                  <div className="extract-body">{currentComment && renderRichtext(currentComment.body)}</div>
-                  <div className="next-extract">
-                    {extracts &&
-                      extractIndex < extracts.length - 1 && (
-                        <div
-                          onClick={() => {
-                            this.changeCurrentExtract(1);
-                          }}
-                        >
-                          <span className="assembl-icon-down-open grey" />
-                        </div>
-                      )}
-                  </div>
-                </div>
-              )}
-            {submitting && (
-              <div className="submit-comment">
-                <RichTextEditor
-                  editorState={body}
-                  maxLength={FICTION_COMMENT_MAX_LENGTH}
-                  onChange={this.updateBody}
-                  placeholder={I18n.t('debate.brightMirror.commentLabel')}
-                  withAttachmentButton
-                />
-              </div>
-            )}
-          </div>
+          )}
+          {submitting ? (
```

Comment:
add a new component ExtractCommentForm ?

Reply:
InnerBoxSubmit has been created for this, to be used for the comments and the replies